package.json

```diff
@@ -33,17 +33,23 @@
     "console-log-level": "^1.4.0",
     "delay": "^4.0.1",
     "extend": "^3.0.1",
+    "findit2": "^2.2.3",
     "gcp-metadata": "^0.9.0",
+    "lodash": "^4.17.10",
```

Comment:
We've found the full scale lodash package to be really, really slow to load.  I'll keep reading, but chances are you could install sub-packages that are far more targeted.